### DISCUSSION TITLE
How to get List of Deployments in namespace?

### DISCUSSION THREAD

**Discussion by nollidnosnhoj** [kubernetes-client/csharp#986] [Q&A] [answered]
How to get a list of deployments within a namespace? I try finding a method for it under CoreV1 but it seems like it was not generated.

**Comment by tg123** [comment, marked as answer]
deployment is under AppsV1 catalog 
https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.25/#deployment-v1-apps

**Reply by nollidnosnhoj** [reply]
This works! Thanks
